src/compaction.c

```diff
@@ -122,7 +122,7 @@ double VarPopulationFinalize(void *contextPtr) {
     if(count <= 1) {
         return 0;
     }
```

Comment:
This test should be handled by `variance` function (before dividing by `count`)

Reply:
Nice catch!
Done.